src/java/main/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -1047,14 +1047,19 @@ public void processConnectRequest(ServerCnxn cnxn, ByteBuffer incomingBuffer) th
         cnxn.disableRecv();
         long sessionId = connReq.getSessionId();
         if (sessionId == 0) {
-            LOG.info("Client attempting to establish new session at "
-                    + cnxn.getRemoteSocketAddress());
-            createSession(cnxn, passwd, sessionTimeout);
+            long id = createSession(cnxn, passwd, sessionTimeout);
+            LOG.info("Client attempting to establish new session: session = 0x"
```

Comment:
I like these ones. Would you please use the message formatter of the logging lib? e.g. `"... {}... {}", a,b`

Reply:
shouldn't this also be debug like the other session messages?